.github/workflows/pull-request-integ.yml

```diff
@@ -28,6 +28,102 @@ jobs:
     name: Integrations Tests
     runs-on: ubuntu-20.04
     if: contains(github.event.pull_request.labels.*.name, 'pr-integrations')
+    env:
+      NESSIE_DIR: included-builds/nessie
```

Comment:
`included-builds` has special meaning, does it not? Could you add a comment about that?

Reply:
It's in the README.md https://github.com/projectnessie/query-engine-integration-tests

Reply:
Yes, but it's a different repo... It would help readers if we had a brief local comment about that :)